tests/openff_md/conftest.py

```diff
@@ -0,0 +1,102 @@
+import tempfile
+from pathlib import Path
+
+import numpy as np
+import openff.toolkit as tk
+import pytest
+from jobflow import run_locally
+from openff.interchange import Interchange
+from openff.interchange.components._packmol import pack_box
+from openff.toolkit import ForceField
+from openff.units import unit
+
+from atomate2.openff.utils import create_mol_spec, merge_specs_by_name_and_smiles
+
+
+@pytest.fixture
+def temp_dir():
+    with tempfile.TemporaryDirectory() as temp_dir:
+        yield Path(temp_dir)
```

Comment:
redundant fixture, [`pytest` ships its own `tmp_path`](https://docs.pytest.org/en/stable/how-to/tmp_path.html#the-tmp-path-fixture)